src/renderer/containers/ReplayBrowser/FileSelectionToolbar.tsx

```diff
@@ -53,13 +57,12 @@ export const FileSelectionToolbar: React.FC<FileSelectionToolbarProps> = ({
           {totalSelected} files selected
         </div>
         <div>
-          <Button
-            color="secondary"
-            variant="contained"
-            size="small"
-            onClick={() => setShowDeletePrompt(true)}
-            startIcon={<DeleteIcon />}
-          >
+          <DraggableFiles fullPaths={filePaths}>
+            <Button color="secondary" variant="contained" size="small" startIcon={<DragIndicatorIcon />}>
```

Comment:
```suggestion
            <Button
              color="secondary"
              variant="contained"
              size="small"
              startIcon={<DragIndicatorIcon />}
              css={css`
                cursor: inherit;
              `}
            >
```

Needs to inherit cursor otherwise you just get a pointer when hovering over the button.